README.md

```diff
@@ -933,16 +1003,176 @@ SUB COMMANDS:
+            Force a device not in BOOTSEL mode but running compatible code to reset so the command can be executed. After executing the
+            command (unless the command itself is a 'reboot') the device will be left connected and accessible to picotool, but without the
+            RPI-RP2 drive mounted
+```
+
+```text
+$ picotool help otp set
+OTP SET:
+    Set the value of an OTP row/field
+
+SYNOPSIS:
+    picotool otp set [-c <copies>] [-r] [-e] [-s] [-i <filename>] [-z] <selector> <value> [device-selection]
+
+OPTIONS:
+    Redundancy/Error Correction Overrides
+        -c <copies>
+            Read multiple redundant values
+        -r, --raw
+            Set raw 24 bit values
+        -e, --ecc
+            Use error correction
+        -s, --set-bits
+            Set bits only
+        -i <filename>
+            Include extra otp definition
+        <value>
+            The value to set
+    Row/Field Selection
+        -z, --fuzzy
+            Allow fuzzy name searches in selector vs exact match
+        <selector>
+            The row/field selector, which can be:
+            ROW_NAME or ROW_NUMBER or PAGE:PAGE_ROW_NUMBER to select a whole row.
+            FIELD, REG.FIELD, REG.n-m, PAGE:PAGE_ROW_NUMBER.FIELD or PAGE:PAGE_ROW_NUMBER.n-m to select a row field.
+
+            where:
+
+            REG and FIELD are names (or parts of names with fuzzy searches).
+            PAGE and PAGE_ROW_NUMBER are page numbers and row within a page, ROW_NUMBER is an absolute row number offset, and n-m are the
+            inclusive bit ranges of a field.
+
+TARGET SELECTION:
+    Target device selection
+        --bus <bus>
+            Filter devices by USB bus number
+        --address <addr>
+            Filter devices by USB device address
+        --vid <vid>
+            Filter by vendor id
+        --pid <pid>
+            Filter by product id
+        --ser <ser>
+            Filter by serial number
+        -f, --force
+            Force a device not in BOOTSEL mode but running compatible code to reset so the command can be executed. After executing the
+            command (unless the command itself is a 'reboot') the device will be rebooted back to application mode
+        -F, --force-no-reboot
+            Force a device not in BOOTSEL mode but running compatible code to reset so the command can be executed. After executing the
+            command (unless the command itself is a 'reboot') the device will be left connected and accessible to picotool, but without the
+            RPI-RP2 drive mounted
+```
+
 ### load
 
 This command allows loading of a range of OTP rows onto the device. The source can be a binary file, or a JSON file such as the one output by `picotool sign`. The schema for this JSON file is [here](json/schemas/otp-schema.json)
+
+```text
+$ picotool help otp load
+OTP LOAD:
+    Load the row range stored in a file into OTP and verify. Data is 2 bytes/row for ECC, 4 bytes/row for raw.
```

Comment:
> 4 bytes/row for raw

Will the 4th byte simply get ignored? Does it ignore the MSB or LSB?

Reply:
It ignores the MSB - I'll add a note to that line